src/library/scala/collection/immutable/HashMap.scala

```diff
@@ -97,17 +98,6 @@ sealed class HashMap[A, +B] extends AbstractMap[A, B]
 
   protected def filter0(p: ((A, B)) => Boolean, negate: Boolean, level: Int, buffer: Array[HashMap[A, B @uV]], offset0: Int): HashMap[A, B] = null
 
-  protected def elemHashCode(key: A) = key.##
```

Comment:
User defined element hashing was removed in 2.13 with a migration suggestion to use use wrapper classes for keys. But I don't think we can remove this in a minor release. Did you intend to include this diff in this PR?

Reply:
I didn't change the functionality her, just moved it to be in the object, as I needed the hashing from the builder

Is your concern the binary compat, or the functionality?
If the former, the methods can remain but be unused?

Reply:
The functionality. Admittedly it does not have test coverage but it is used a few times in the wild: https://github.com/search?q=%22override+def+elemHashCode%22&type=Code

Reply:
i have restored the extension point